cli/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 _Released 11/5/2024 (PENDING)_
 
+**Features:**
+
+- Added new [`defaultBrowser`](https://docs.cypress.io/app/references/configuration) configuration option to specify the default browser to launch. Addresses [#6646](https://github.com/cypress-io/cypress/issues/6646).
```

Comment:
It'd be great to specify the behavior of  what this will functionally do in open vs run mode here.

Reply:
@jennifer-shehane do you mean in the changelog or the documentation https://github.com/cypress-io/cypress-documentation/pull/5991 ?